package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve --proxy-config conf/proxy.conf.json",
+    "start": "ng serve --public-host 4200-${GITPOD_WORKSPACE_ID}.${GITPOD_WORKSPACE_CLUSTER_HOST} --proxy-config conf/proxy.conf.json",
```

Comment:
Does this affect things when starting this locally?

Reply:
no, apart from a very specific minor version of Node 19. With 19.6.0 this does cause issues, but with newer 19s or versions 16.x.x this doesn't.